lib/lita/handlers/jira_issues.rb

```diff
@@ -80,6 +82,26 @@ def priority(data)
       def issue_link(key)
         "\n#{config.url}/browse/#{key}"
       end
+
+      def silenced?(key)
+
+        if config.issue_ttl == 0
+          log.debug("JiraIssues: issue_ttl is set to 0, will post every matched issue to chat")
+          return false
+        end
+
+        current_ttl = redis.ttl(key).to_i
+
+        if current_ttl > 0
+          log.debug("JiraIssues: Key expiration not met for #{key}, will not reprompt for #{current_ttl} seconds")
+          true
+        else
+          redis.setex(key, config.issue_ttl, key)
```

Comment:
I guess another reason I'm not a huge fan of log statements is I miss important code like this. I find myself subscribing to Avdi Grimm's [Confident Ruby](http://www.confidentruby.com/) approach of putting guard statements at the top in single like if methods which leaves important code like this at the closes indentation level. One approach to this method might look something like:

```ruby
def silenced?(key)
  return if config.issue_ttl == 0
  
  current_ttl = redis.ttl(key).to_i
  return true if current_ttl > 0

  redis.setex(key, config.issue_ttl, key)
  false
```

Anyway, not saying that's the right way, but it does avoid indentation which Avdi argues makes code less "confident"